bin/deploy-lambda.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+isNpmPackageInstalled() {
+  which node-lambda $1 > /dev/null 2>&1
+}
+
+for package in node-lambda
+do
+  if ! ( isNpmPackageInstalled $package ); then
+      echo $package is NOT installed
+      npm install -g node-lambda
+  fi
+done
+
```

Comment:
Instead of 
```
isNpmPackageInstalled() {
  which node-lambda $1 > /dev/null 2>&1
}

for package in node-lambda
do
  if ! ( isNpmPackageInstalled $package ); then
      echo $package is NOT installed
      npm install -g node-lambda
  fi
done
```

please use: 

```bash
if [[ -z $(which node-lambda) ]]; then 
    echo "Installing node-lambda globally..."
    npm install -g node-lambda
fi
```

**P.S. the less code - the less bugs**